integration-tests/package.json

```diff
@@ -22,7 +22,7 @@
     "google-auth-library": "^7.9.2",
     "lodash": "^4.17.21",
     "p-retry": "^4.6.1",
-    "puppeteer": "22.7.1",
+    "puppeteer": "22.11.2",
```

Comment:
puppeteer also has a transitive dependency to a vulnerable version of `ws`, so I bumped puppeteer up too.

I am not sure why puppeteer is pinned to an exact version instead of using `^` or `~`, but I maintained that choice here.